interface/super/rules/www/css/rules.css

```diff
@@ -102,10 +102,6 @@ root {
   display: block;
 }
 
-.rule_detail p {
```

Comment:
I also had to remove this negative margin and I didn't see issues on other forms, I tested it on both Chrome and Firefox on Linux but it might be a good idea to check it out in another OS be it OSX or Windows.